packages/api/src/routes/medical/schemas/address.ts

```diff
@@ -1,9 +1,7 @@
 import { z } from "zod";
-import { USState } from "@metriport/core/domain/geographic-locations";
+import { usStateSchema } from "@metriport/api-sdk/medical/models/common/us-data";
```

Comment:
Great that you're updating this. Since we're here, could you also check the other `usStateSchema` we have on the API and maybe refer to the same place, please?

Reply:
Only one other place to update, `driverLicenseIdentifierSchema` which I updated